integration_tests/benchmarks/benchmark.html

```diff
@@ -193,19 +246,20 @@ <h2>TensorFlow.js Model Benchmark</h2>
       tbody.appendChild(tr);
     }
 
+    async function awaitResult(res) {
+      if (res instanceof Promise) {
+        return await res;
```

Comment:
if the returen type is Promise<Tensor>, you would need call await (await res).dataSync() ?